specification/cognitiveservices/data-plane/SpeechServices/stable/v2.0/Speech.json

```diff
@@ -1792,46 +1792,7 @@
           "200": {
             "description": "OK",
             "schema": {
-              "title": "IReadOnlyDictionary`2",
-              "type": "object",
-              "properties": {
-                "None": {
-                  "type": "array",
-                  "items": {
-                    "type": "string"
-                  }
-                },
-                "Language": {
-                  "type": "array",
-                  "items": {
-                    "type": "string"
-                  }
-                },
-                "Acoustic": {
-                  "type": "array",
-                  "items": {
-                    "type": "string"
-                  }
-                },
-                "Pronunciation": {
-                  "type": "array",
-                  "items": {
-                    "type": "string"
-                  }
-                },
-                "CustomVoice": {
-                  "type": "array",
-                  "items": {
-                    "type": "string"
-                  }
-                },
-                "LanguageGeneration": {
-                  "type": "array",
-                  "items": {
-                    "type": "string"
-                  }
-                }
-              }
+              "$ref": "#/definitions/IReadOnlyDictionary`2"
```

Comment:
is the "`" an intentional character in this name? When generating code, this name will turn into a Class name.